lib/live_view_native/jetpack/rules_parser/modifiers.ex

```diff
@@ -274,6 +265,11 @@ defmodule LiveViewNative.Jetpack.RulesParser.Modifiers do
         frozen(parsec(:nested_modifier)),
         "a modifier eg ‘bold()’"
       },
+      {
+        # Must come after :ime and :nested_modifier parsers
+        literal(error_parser: empty(), generate_error?: false),
+        ~s'a number, string, nil, boolean or :atom'
+      },
```

Comment:
Move the literal parser to be after the `nested_modifier` parser.

`Dp(1)` was eagerly being treated as an atom `:Dp` followed by `(1)`. It should instead be treated as a nested modifier.